components/EditISCNMetadataModal.vue

```diff
@@ -149,6 +149,10 @@ watchEffect(async () => {
               ]
 
           iscnData.value = {
+            ...metadata,
+            id: record.id,
+            ipld: record.ipld,
```

Comment:
Can safely skip these 2 since they are calculated
On the other hand we might need `recordNotes`